esx_service/vmdk_ops.py

```diff
@@ -696,20 +696,24 @@ def get_vol_path(datastore, tenant_name=None):
     # is created on <datastore>/DOCK_VOLS_DIR/tenant_uuid
     # a symlink <datastore>/DOCK_VOLS_DIR/tenant_name will be created to point to
     # path <datastore>/DOCK_VOLS_DIR/tenant_uuid
+    # If the command is under a tenant then the path returned contains tenant
```

Comment:
This is just an extension of the already present comment at the start of this function mentioned below:

"# If the command is NOT running under a tenant, the folder for Docker
 # volumes is created on <datastore>/DOCK_VOLS_DIR"

By this the original commenter means that tenant_name is not none in the args.